requirements.txt

```diff
@@ -0,0 +1,26 @@
+dateparser
+django~=3.2.14
+django-hstore
+django-markdownx @ git+https://github.com/alphatownsman/django-markdownx.git@e69480c64ad9c5d0499f4a8625da78cf2bb7691b
+django-sass
+django-rq
+django-simple-history
+django-hijack
+django-user-messages
+django-slack
+meilisearch
+easy-thumbnails
+lxml
+openpyxl
+psycopg2
+requests
+filetype
+setproctitle
```

Comment:
I don't find the occurrence of this package in the code, was this an attempt to change the process name or get the name of some process?

Reply:
`django-rq` worker process will have a better title (as displayed by system tools such as ps and top).